code/worker.py

```diff
@@ -1,38 +1,40 @@
 #!/usr/bin/env python3
+import argparse
+
+import requests
+
+
+def main() -> None:
+    parser = argparse.ArgumentParser()
+    parser.add_argument(
+        'appimage_url', metavar='URL',
+        help="URL of the AppImage file to query")
+    ns = parser.parse_args()
+    r = requests.get(ns.appimage_url, stream=True)
```

Comment:
You should certainly improve your variable naming here. Maybe even insert a line before.